paper_on_queues_and_stack_structures.md

```diff
@@ -0,0 +1,41 @@
+# Queues and Stack Structures  
+Queues and stack structures are data structures that differ when it comes to the modification of data. 
+The principle for queues is the data that is first in, gets out first. For stacks, the principle is last in, first out. 
+A good visualization would be that a queue is a rectangular data structure that manages data from left to right or vice versa, such as a line of people, while a stack is structured similarly to a stack of paper that is managed starting from the top. 
+Queues and stack structures have their own trade-offs, but they both play vital roles in the function of computer science.
+
+### Introduction to Queues
```

Comment:
I believe it would look simpler and cleaner if the title for this section were just "Queues" instead of "Introduction to Queues".